src/device_stm32h5.c

```diff
@@ -12,6 +12,8 @@
 #define FLASH_NSCCR (FLASH_BASE + 0x30)
 #define FLASH_OPTSR_CUR (FLASH_BASE + 0x50)
 #define FLASH_OPTSR_PRG (FLASH_BASE + 0x54)
+#define SCB (0xE000E000UL +  0x0D00UL)
```

Comment:
That's a constant, why not this:
```c
#define SCB 0xe000ed00   // Base address of a System Control Block
```

Reply:
CMSIS stuff, SCB->VTOR, present in all ARM cores except M0 and M0+ where it is optional (IIRC) and there is an SCB_VTOR_PRESENT constant or something like that to check for it